extensions/clickup/package.json

```
@@ -1,18 +1,26 @@
 {
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "manage-clickup-tasks",
-  "title": "Clickup Tasks Explorer",
+  "title": "Tasks Explorer",
   "description": "Browse ClickUp tasks from Raycast",
   "icon": "clickup-icon.png",
   "author": "aparandeh",
+  "contributors": [
+    "aesadde"
+  ],
   "license": "MIT",
   "commands": [
     {
-      "name": "index",
-      "title": "Tasks Explorer",
-      "subtitle": "Clickup",
+      "name": "explore",
+      "title": "Clickup Tasks Explorer",
```

```suggestion
      "title": "Tasks Explorer",
      "subtitle": "ClickUp",
```
Should be a better naming